website/content/en/docs/kubebuilder-migration/project_migration_guide.md

```diff
@@ -0,0 +1,178 @@
+# Overview - Single API version operator
+
+This guide walks through the steps required to migrate an operator from an Operator SDK project layout to a Kubebuilder project layout.
+
+The document considers [memcached operator][memcached-operator] as an example to describe the process of migrating an operator built using Operator SDK to a runnable Kubebuilder project.
+
+## Prerequisites
+- [git][git_tool].
+- [go][go_tool] version 17.03+.
+- [kubectl][kubectl_tool] version v1.11.3+.
+- [kustomize][kustomize_tool] v3.1.0+.
+- Access to a Kubernetes v1.11.3+ cluster.
+
```

Comment:
Should we not ask for the user to ensure that their project is upgraded to the latest version of SDK first?